app/models/manageiq/providers/amazon/network_manager/refresh_parser_dto.rb

```diff
@@ -0,0 +1,575 @@
+class ManageIQ::Providers::Amazon::NetworkManager::RefreshParserDto
+  include ManageIQ::Providers::Amazon::RefreshHelperMethods
+
+  def initialize(ems, options = nil)
+    @ems        = ems
+    @aws_ec2    = ems.connect
+    @aws_elb    = ems.connect(:service => :ElasticLoadBalancing)
+    @data       = {:_dto_collection => true}
+    @data_index = {}
+    @options    = options || {}
+    initialize_dto_collections
+  end
+
+  def add_dto_collection(model_class, association, manager_ref = nil)
```

Comment:
Let's get an issue in place in MIQ repo for changing this name.  I don't like "DTO Collection" here.

I don't have a great suggestion off the top of my head yet, but I want it to be more "inventory" or "refresh" specific.

(does not hold up merging)